flexmeasures/ui/templates/defaults.jinja

```diff
@@ -11,7 +11,6 @@
 {% set nav_bar_specs = {
     "dashboard": dict(title="Dashboard", icon="dashboard"),
     "assets": dict(title="Assets", icon="list-ul"),
```

Comment:
I feel assets can go, they are available from accounts.

Reply:
I agree with Nicolas, but with the following clarification: I would only remove this page from the menu **for non-admin users**, who already get to see the same table when they click on the "My account" menu item. For admin users, I would **not** remove this page from the menu, because it is the only place where a big table with all assets across all accounts is shown. (And I use it a lot.)

Reply:
Assets and Users will only show in the navbar for Admins